spec/jobs/ingest_archival_media_bag_job_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe IngestArchivalMediaBagJob do
+  describe 'lae bag' do
+    with_queue_adapter :inline
+    let(:bag_path) { Rails.root.join('spec', 'fixtures', 'av', 'la_c0652_2017_05_bag') }
+    let(:xml) { File.open(Rails.root.join('spec', 'fixtures', 'av', 'C0652.xml'), 'r') }
+    let(:user) { FactoryBot.create(:admin) }
+
+    let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+    let(:storage_adapter) { Valkyrie.config.storage_adapter }
+    let(:query_service) { adapter.query_service }
+    let(:change_set_persister) { PlumChangeSetPersister.new(metadata_adapter: adapter, storage_adapter: storage_adapter) }
+    let(:collection_cid) { collection.source_metadata_identifier.first }
+    # We need to actually save it with persister to get the imported metadata
+    let(:collection) do
+      cs = DynamicChangeSet.new(ArchivalMediaCollection.new)
+      cs.validate(source_metadata_identifier: "C0652", bag_path: bag_path)
+      cs.sync
+      change_set_persister.save(change_set: cs)
+    end
+
+    before do
+      stub_request(:get, "https://findingaids.princeton.edu/collections/C0652.xml?scope=record")
```

Comment:
Small thing, but can you just use `stub_pulfa` from https://github.com/pulibrary/figgy/blob/master/spec/support/stub_bibdata.rb#L20 ?